packages/MorphicAPIExplorer-Core.package/MEXToolOverView.class/instance/setSelectionFor..st

```diff
@@ -0,0 +1,5 @@
+accessing
+setSelectionFor: aMethod
```

Comment:
aMethod ist kein idealer Name, weil es ja auch eine category sein kann, auch der methodenname beschreibt nicht, was getan wird, vielleicht eher switchToExplorationViewFor

Reply:
setSelectionFor macht Sinn, weil es der setter in dem Baum ist